<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolIconMenuMainWnd</class>
 <widget class="QWidget" name="ToolIconMenuMainWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="ToolIconMenuMainWndLabel" name="labelPic">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>60</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50, 50, 50);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clearTaskBtn">
   <property name="geometry">
    <rect>
     <x>357</x>
     <y>551</y>
     <width>75</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stepDeleteBtn">
   <property name="geometry">
    <rect>
     <x>267</x>
     <y>551</y>
     <width>75</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LearnOneRound">
   <property name="geometry">
    <rect>
     <x>447</x>
     <y>551</y>
     <width>75</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>取样</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrevPage">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>615</y>
     <width>70</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>|&lt;</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNextPage">
   <property name="geometry">
    <rect>
     <x>842</x>
     <y>615</y>
     <width>70</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;|</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="toolViewBtn">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>551</y>
     <width>75</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QListWidget" name="taskListWidget">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>616</y>
     <width>750</width>
     <height>140</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="movement">
    <enum>QListView::Static</enum>
   </property>
   <property name="flow">
    <enum>QListView::TopToBottom</enum>
   </property>
   <property name="isWrapping" stdset="0">
    <bool>true</bool>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="viewMode">
    <enum>QListView::ListMode</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="stepEditBtn">
   <property name="geometry">
    <rect>
     <x>177</x>
     <y>551</y>
     <width>75</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </widget>
  <widget class="QLabel" name="stepTimeTitleLabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>6</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>单步时间：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="oneRoundTimeLabel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>36</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>----.-- ms</string>
   </property>
  </widget>
  <widget class="QLabel" name="stepNameLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>6</y>
     <width>250</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-.----</string>
   </property>
  </widget>
  <widget class="QLabel" name="stepStatusLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>6</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>144</red>
         <green>144</green>
         <blue>144</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>144</red>
         <green>144</green>
         <blue>144</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>144</red>
         <green>144</green>
         <blue>144</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>144</red>
         <green>144</green>
         <blue>144</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>144</red>
         <green>144</green>
         <blue>144</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>144</red>
         <green>144</green>
         <blue>144</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>144</red>
         <green>144</green>
         <blue>144</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>144</red>
         <green>144</green>
         <blue>144</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>144</red>
         <green>144</green>
         <blue>144</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(144, 144, 144);</string>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="judgeTitleLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>6</y>
     <width>40</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>综合
判定</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="oneRoundTimeTitleLabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>36</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>单轮时间：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="stepExecuteTimeLabel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>6</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>----.-- ms</string>
   </property>
  </widget>
  <widget class="StepExcuteLabel" name="labelExcuteData">
   <property name="geometry">
    <rect>
     <x>648</x>
     <y>63</y>
     <width>280</width>
     <height>431</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDataPre">
   <property name="geometry">
    <rect>
     <x>668</x>
     <y>499</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtPointNum">
   <property name="geometry">
    <rect>
     <x>803</x>
     <y>508</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDataNext">
   <property name="geometry">
    <rect>
     <x>858</x>
     <y>499</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>508</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtCurrentPoint">
   <property name="geometry">
    <rect>
     <x>735</x>
     <y>508</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StepExcuteLabel</class>
   <extends>QLabel</extends>
   <header>Tool/MainPanel/Roi/StepExcuteLabel.h</header>
  </customwidget>
  <customwidget>
   <class>ToolIconMenuMainWndLabel</class>
   <extends>QLabel</extends>
   <header>Tool/MainPanel/Roi/ToolIconMenuMainWndLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
